warehouse/query-core/src/test/java/datawave/query/util/MockMetadataHelper.java

```diff
@@ -18,7 +18,6 @@
 import datawave.query.model.QueryModel;
 import datawave.util.TableName;
 import org.apache.accumulo.core.client.AccumuloClient;
-import org.apache.accumulo.core.client.AccumuloException;
```

Comment:
MarkingFunctions import above is unused.

Reply:
Removed